third_party/2/google/protobuf/struct_pb2.pyi

```diff
@@ -0,0 +1,103 @@
+from google.protobuf.internal.containers import (
+    RepeatedCompositeFieldContainer,
+)
+from google.protobuf.message import (
+    Message,
+)
+from typing import (
+    Iterable,
+    List,
+    Mapping,
+    MutableMapping,
+    Optional,
+    Text,
+    Tuple,
+    cast,
+)
+
+
+class NullValue(int):
+    @classmethod
+    def Name(cls, number: int) -> str: ...
+
+    @classmethod
+    def Value(cls, name: str) -> NullValue: ...
+
+    @classmethod
+    def keys(cls) -> List[str]: ...
+
+    @classmethod
+    def values(cls) -> List[NullValue]: ...
+
+    @classmethod
+    def items(cls) -> List[Tuple[str, NullValue]]: ...
+
+
+NULL_VALUE = cast(NullValue, 0)
```

Comment:
Can you change this to `NULL_VALUE: NullValue`? I don't think `cast` is supposed to be present in stubs, even if type checkers currently support it.